lm_eval/models/huggingface.py

```diff
@@ -150,8 +111,7 @@ def __init__(
             gpus = torch.cuda.device_count()
             accelerator_kwargs = InitProcessGroupKwargs(timeout=timedelta(weeks=52))
             accelerator = Accelerator(kwargs_handlers=[accelerator_kwargs])
-            if accelerator.num_processes > 1:
-                self.accelerator = accelerator
+            self.accelerator = accelerator
```

Comment:
added this because I use accelerator to wait for processes when dowloading models